presto-docs/src/main/sphinx/connector/memory.rst

```diff
@@ -0,0 +1,67 @@
+================
+Memory Connector
+================
+
+The Memory connector stores all data and metadata in RAM on workers
+and both are discarded when Presto restarts.
+
+.. warning::
+
+    This connector is in early experimental stage it is not recommended
+    to use it in a production environment. The intended use of this
+    connector at this point is purely for testing.
+
+.. warning::
+
+    After ``DROP TABLE`` memory is not released immediately. It is released
+    after next write access to memory connector.
```

Comment:
It would require both sending `activeTableIds` and add some (small) overhead for checking it and dropping pages on every read access.

On the other hand it doesn't seem to brake anything. Administrator/user must configure this connector and cluster by taking into account maximum potential memory usage by this connector and can't relay that this connector will temporarily use less memory. So it shouldn't matter if the memory is released on first read or first write access.

However I'm not strongly in favor of either version